src/index.html

```diff
@@ -9,10 +9,58 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/style.css"
     />
```

Comment:
Keep your attributes correctly formatted. Since this `link` element has more than 2 attributes, each attribute should be on a new line with 2-space indentation related to the tag.